coral-common/src/main/java/com/linkedin/coral/common/calcite/CalciteUtil.java

```diff
@@ -123,4 +127,9 @@ public static String quoteReservedWords(String s) {
     s = s.replaceAll("(^|\\W)rank($|\\W)", "$1\"rank\"$2");
     return s;
   }
+
+  public static SqlOperator createSqlOperatorOfFunction(String functionName, SqlReturnTypeInference typeInference) {
```

Comment:
nit: can we simplify the name to "createSqlOperator"?

Reply:
per the discussion, this function name will be kept for now.